src/Okta.Sdk/Internal/HomePath.cs

```diff
@@ -47,28 +50,43 @@ public static string Resolve(params string[] pathSegments)
 
         /// <summary>
         /// Resolves the current user's home directory path.
+        /// Throws an exception if the path cannot be resolved.
         /// </summary>
         /// <returns>The home path.</returns>
         public static string GetHomePath()
         {
```

Comment:
Is this method still necessary?

Reply:
This one is declared as public so this would be breaking changes if we remove it. 
I asked Bryan about this and he said if we decide to delete a public function, we need to follow a deprecation procedure (obsolete things for deletion in a later release).
Should we do this, what do you think?

Reply:
I hope no one is using this since this is in the `Internal` namespace. However, let's keep it just in case and we can remove it in the next major version.